exporter/opentelemetry-exporter-jaeger/src/opentelemetry/exporter/jaeger/__init__.py

```diff
@@ -222,10 +219,14 @@ def _collector_http_client(self) -> Optional[Collector]:
         return self._collector
 
     def export(self, spans) -> SpanExportResult:
+
+        tracer_provider = trace.get_tracer_provider()
+        resource = tracer_provider.resource
+        svc_name = resource.attributes[SERVICE_NAME]
```

Comment:
I think you don't need to do this every time export is called. You can just set this we Exporter instance is created and use it.